projects/sky-autonumeric/src/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -63,11 +63,11 @@ export class SkyAutonumericDirective
     this.updateAutonumericInstance();
   }
 
-  private autonumericInstance: AutoNumeric;
-  private autonumericOptions: SkyAutonumericOptions;
-  private control: AbstractControl;
+  private autonumericInstance!: AutoNumeric;
+  private autonumericOptions!: SkyAutonumericOptions;
+  private control?: AbstractControl;
```

Comment:
Let's talk about the difference between `?` and `| undefined` on class properties. I thought there was going to be a difference in the compiled output, but they seem to be equivalent for classes; however they're not equivalent for interfaces or types, so we need to decide whether we care to try to be consistent across all three scenarios.